models/migrations/migrations.go

```diff
@@ -388,9 +388,11 @@ var migrations = []Migration{
 	// v215 -> v216
 	NewMigration("allow to view files in PRs", addReviewViewedFiles),
 	// v216 -> v217
-	NewMigration("Improve Action table indices", improveActionTableIndices),
+	NewMigration("(NOOP) Improve Action table indices", noop216),
```

Comment:
Given that reverted migrations have already occurred more than once, what about defining a general `noop` function to use instead of redefining it every single time?